api.rst

```diff
@@ -651,6 +651,49 @@ Embedded resources can be aliased and filters can be applied on these aliases:
 
   GET /films?select=*,90_comps:competitions(name),91_comps:competitions(name)&90_comps.year=eq.1990&91_comps.year=eq.1991 HTTP/1.1
 
+.. _embedding_partitioned_tables:
+
+Embedding Partitioned Tables
+----------------------------
+
+Embedding can also be done between partitioned tables and other resources.
```

Comment:
```suggestion
Embedding can also be done between partitioned tables and other tables.
```

Reply:
Also, I think it would be nice to link to partition docs here: https://www.postgresql.org/docs/current/ddl-partitioning.html